WearStandaloneGoogleSignIn/app/src/main/res/values/strings.xml

```diff
@@ -1,24 +1,22 @@
-<!--
-  ~ Copyright 2016 Google Inc.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~      http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-<resources>
+<!-- Copyright (C) 2021 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_name">googlesignin</string>
-    <!-- Long form sign-in button text [CHAR LIMIT=30] -->
     <string name="wear_signout_button_text">Sign out</string>
     <string name="google_signin_successful">Google Sign-In successful!</string>
     <string name="signout_successful">Successfully signed out!</string>
     <!-- Used by the Google Sign In API for auth. -->
-    <string name="server_client_id">TODO_ADD_CLIENT_ID_XXXXXXX.apps.googleusercontent.com</string>
+    <string name="server_client_id" tools:ignore="UnusedResources">TODO_ADD_CLIENT_ID_XXXXXXX.apps.googleusercontent.com</string>
```

Comment:
@codingjeremy Should this value be used somewhere in the app?

Reply:
Investigated some more: Registering the app's package name and signature is necessary, but this `server_client_id` doesn't do anything in the strings. In the latest commit, I removed it.

Reply:
SG, just to verify, you checked this with a live project (where you registered them), correct?

I used to have to include that for an actual project but that's great if it is no longer included.

I thought I had responded to this but maybe we discussed on chat? ^_^;;

Reply:
Yes, I did! Just registering the package name and the APK signature was enough, didn't have to include anything in `strings.xml`.

Reply:
SGTM, thanks!